src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    public String getRandomColor() {
+        int index = new Random().nextInt(Color.values().length);
+        String color = Color.values()[index].toString();
+        return color;
```

Comment:
1. inline this 
2. https://github.com/mate-academy/jv-oop-advanced/blob/master/checklist.md#use-name-for-getting-string-representation-of-enum-constants
```suggestion
        String color = Color.values()[index].toString();
        return color;
```